docs/README.md

```diff
@@ -199,11 +199,11 @@ The `--dts-only` flag is the equivalent of the `emitDeclarationOnly` option in `
 
 #### Generate TypeScript declaration maps (.d.ts.map)
 
-TypeScript declaration maps are mainly used to quickly jump to type definitions in the context of a monorepo (see [source issue](https://github.com/Microsoft/TypeScript/issues/14479) and [official documentation](https://www.typescriptlang.org/tsconfig/#declarationMap)).
+TypeScript declaration maps are mainly used to quickly jump to the source code associate with imported objects. Without declaration maps, `Go to Definition` in IDEs will go to the type defition and not the relevant source code. `Go to Type Definition` will always go to the type definition.
 
-They should not be included in a published NPM package and should not be confused with sourcemaps.
+Declaration maps and source TS files should be included in all published TypeScript packages.
```

Comment:
```suggestion
Declaration maps and source TS files should be included in all TypeScript packages.
```

They're also useful in monorepos, as the original version stated.

Reply:
Does make sense.

I am partial to using `additional conditions` to have my local packages point directly at the TS source of my monorepo packages, but makes sense.